internal/cli/kraft/remove/remove.go

```diff
@@ -229,7 +229,11 @@ func (opts *RemoveOptions) Run(ctx context.Context, args []string) error {
 		}
 
 		// Stop the machine before deleting it.
-		if machine.Status.State == machineapi.MachineStateRunning {
+		if machine.Status.State == machineapi.MachineStateRunning ||
+			machine.Status.State == machineapi.MachineStateCreated ||
+			machine.Status.State == machineapi.MachineStatePaused ||
+			machine.Status.State == machineapi.MachineStateRestarting ||
+			machine.Status.State == machineapi.MachineStateSuspended {
```

Comment:
Is the if-statement even necessary in fact? The error is just logged and not returned.

Reply:
depends on how you like it, take the case of running `rm --all` after a long time

You will be spammed by 20-30 error messages, when in fact all is good cause you want those removed

Reply:
What about attempting to stop it and then logging the error in case the State is not Exited?

Reply:
What happens if the instance is stopped and the `Stop` method is called again? I.e. does it error? If it does not error, we should just call this method regardless as it acts a safety mechanism for ensuring the machine is shutdown.

Reply:
removed if case